### PR TITLE
709 prop types removal

### LINKED ISSUE #709: Remove Prop-Types ★
Just search for prop-types in the code base & keep removing it occurrence as we are migrating to typescript we don't to make use of prop-Types anymore.

You can even a send a PR by removing it in any one of the the file. 🎯

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fixes #709 
Removal of prop-types from the following components:
team.tsx
Login.jsx
SignUp.tsx
JobConfirmation.jsx
dashBoardHeader.js

@divyanshu-rawat Please check

### CODE REVIEW

File: src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx [hunk 1]
```diff
@@ -79,12 +78,12 @@ const Signup = ({signUpRequest, studentSignUpData}) => {
   );
 };
 
-Signup.propTypes = {
-  signUpRequest: PropTypes.func.isRequired,
-  studentSignUpData: PropTypes.shape({
-    isFetching: PropTypes.bool,
-    success: PropTypes.bool,
-    msg: PropTypes.string,
+Signup = {
```
Comment (divyanshu-rawat) [inline src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: Please remove prop-Types.

File: src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx [hunk 2]
```diff
@@ -79,12 +78,12 @@ const Signup = ({signUpRequest, studentSignUpData}) => {
...
+  signUpRequest: func.isRequired,
+  studentSignUpData: shape({
```
Comment (divyanshu-rawat) [inline src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: same here.

File: src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx [hunk 3]
```diff
@@ -79,12 +78,12 @@ const Signup = ({signUpRequest, studentSignUpData}) => {
...
+    isFetching: Boolean,
+    success: Boolean,
+    msg: String,
   }).isRequired,
```
Comment (divyanshu-rawat) [inline src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: same here.

**Comment by shreyakapoor08** [comment]
@divyanshu did it by mistake i will take the clone again and after making changes send a new PR

File: src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx (hunk 3 above)
Comment (shreyakapoor08) [reply src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: in this i have removed

Comment (shreyakapoor08) [reply src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: did i do something wrong?

Comment (divyanshu-rawat) [reply src/ignitus-Authentication/ignitus-StudentSignUpFlow/ignitus-StudentSignUp/Components/SignUp.tsx]: Yes You have removed the prop-types here.

```
Signup = {
  signUpRequest: func.isRequired,
  studentSignUpData: shape({
    isFetching: Boolean,
    success: Boolean,
    msg: String,
  }).isRequired,
};
```

**Comment by divyanshu-rawat** [comment]
@shreyakapoor08 I am closing this then :)

**Pull request closed without merging**